about.html

```diff
@@ -0,0 +1,119 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Global Leadership Summit-About</title>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.1/css/all.min.css">
+    <link rel="stylesheet" href="./assets/css/header.css">
+    <link rel="stylesheet" href="./assets/css/about.css">
+</head>
+<body>
+    <header>
+        <div class="tab-1">
+            <ul class="tab-1-wrapper">
+                <li><a href="#"><i class="fa-brands fa-facebook-f"></i></a></li>
+                <li><a href="#"><i class="fa-brands fa-twitter"></i></a></li>
+                <li><a href="#">English</a></li>
+                <li><a href="#">My Page</a></li>
+                <li><a href="#">Logout</a></li>
+            </ul>
+        </div>
+
+        <div class="tab-2">
+            <div class="tab-2-wrapper">
+                <div class="nav">
+                    <div class="hamburger">
+                        <i class="fa-solid fa-bars"></i>
+                    </div>
+                    <div class="logo">
+                        <a href="index.html"><img src="./assets/images/main-images/logo.png" alt=""></a>
+                    </div>
+                </div>
+                <ul class="navbar">
+                    <li class="closebar"><i class="fa-solid fa-xmark"></i></li>
+                    <li><a href="index.html">Home</a></li>
+                    <li><a class="active" href="about.html">About</a></li>
+                    <li><a href="#">Program</a></li>
+                    <li><a href="#">Join</a></li>
+                    <li><a href="#">Sponsor</a></li>
+                    <li><a href="#">GL Campaign</a></li>
+                </ul>
+            </div>
+        </div>
+    </header>
+    <main>
```

Comment:
Great job @chibunduonyeje1 🤝

- [ ] As stated in the project requirements, kindly follow the design guideline as much as possible to make your project a close replica of the design provided. You may make sure the background of the first section of your design has a whitish background instead of a dark background.
- Your design.
![your-design](https://user-images.githubusercontent.com/63917725/212492090-dc0298ee-6102-4bcc-ae3f-7cd4fffacdde.PNG)
- Given design
![given-design](https://user-images.githubusercontent.com/63917725/212492109-5d620deb-5171-4905-9b92-e2f889d40e76.PNG)

- [ ] You may also remove the logo from the mobile versions of your design to replicate the design provided.